### ISSUE TITLE
Multiple Server Connections

### ISSUE THREAD

**Issue by pandaboy6621** [leinelissen/jellyfin-audio-player#201]
I would like to be able to switch between different servers. It would be nice if it changes the active one based on what wifi network the device is connected to. Ideally, I want the app to scan for alternate servers in a ranked, re-orderable priority list if one server is unavailable.

**Comment by leinelissen** [comment]
Hey there! This is a good idea, if relatively niche for most users. Hence it won't have priority from me, but if you'd like to take a stab at this, let me know!

**Comment by vijayvcm** [comment]
Yes. This is a much-needed feature for Jellyfin users as Jellyfin doesn’t have a centralized server for managing the IP addresses like how Plex does currently. We often need to switch between the server from LAN to VPN. Typing in the IP address everytime when I change the network is a little cumbersome. So, it would be better if we have a dropdown in place of a text box for choosing an IP address. I don’t think it will take much effort but it would be nice to have a feature.

**Comment by b0081** [comment]
Same here. For me, i don't need the additional features like automatic server change and/or re-scans. But the multi server account (like available in the official android jellyfin app) feature would really be a nice one.
